src/main/java/net/pms/configuration/DeviceConfiguration.java

```diff
@@ -85,7 +85,6 @@ public void inherit(RendererConfiguration ref) throws ConfigurationException {
 		filter = baseConf.filter;
 
 		// Initialize our internal RendererConfiguration vars
-		renderCache = new HashMap<>();
 		sortedHeaderMatcher = ref.sortedHeaderMatcher;
```

Comment:
@SubJunk 😊 

I "read" "Initialize our internal renderer cache". Don't ask me how I managed.... forget it.